thinc/initializers.py

```diff
@@ -29,7 +54,6 @@ def configure_glorot_uniform_init() -> Callable[[Shape], FloatsXd]:
 def zero_init(ops: Ops, shape: Shape) -> FloatsXd:
     return ops.alloc(shape)
 
-
```

Comment:
As an FYI, we use the `black` formatting tool in Thinc and in spaCy. I believe in this case it would keep these additional lines: it has 2 empty lines between functions declared at the top level. Could you install `black` in your IDE to make sure we're not introducing unnecessary formatting changes in PR's ? cf https://black.readthedocs.io/en/stable/